tests/unit/inference/test_inference.py

```diff
@@ -341,6 +341,46 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+# Setup for these models is different from other pipelines, so we add a separate test
+@pytest.mark.inference
```

Comment:
We don't want this running on `inference` workflows, let's make a new `pytest.mark.stable_diffusion`.

Can we also move the test to a new file `test_stable_diffusion.py`?

Reply:
Updated both items, thanks.